tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderEntity.cs

```diff
@@ -63,7 +63,7 @@ public sealed class OrderEntity : EntityBase
                 @LastUpdated,
                 @LastUpdatedBy,
                 @LastUpdatedByName
-            )";
+            );";
```

Comment:
It looks like this file has mixed line endings too – can you fix them?

Reply:
👍